packages/js/plugins/uri-resolvers/ens-resolver/README.md

```diff
@@ -37,7 +37,7 @@ export async function foo({
   client = new PolywrapClient({
     plugins: [
       {
-        uri: "wrap://ens/wraps.eth:ethereum@1.0.0",
+        uri: "wrap://ens/wraps.eth:ethereum@1.1.0",
```

Comment:
i think this readme is outdated right?

Reply:
It is, yeah. We will remove the ens resolver from the repo soon, anyway.